pymongo/pool.py

```diff
@@ -1251,7 +1314,7 @@ class PoolState:
 # Do *not* explicitly inherit from object or Jython won't call __del__
 # http://bugs.jython.org/issue1057
 class Pool:
-    def __init__(self, address, options, handshake=True):
+    def __init__(self, address: Tuple[str, int], options: PoolOptions, handshake: bool = True):
```

Comment:
address should be the Address type alias.

Reply:
ok!